spark-cassandra-connector/src/main/scala/org/apache/spark/sql/cassandra/CassandraSQLContext.scala

```diff
@@ -36,199 +31,48 @@ import collection.mutable
-  }
-
-  /** Add table level write configuration settings. Set cluster to None for a single cluster */
-  def addTableWriteConf(keyspace: String, table: String, conf: WriteConf, cluster: Option[String]) = {
-    cluster match {
-      case Some(c) => validateClusterName(c)
-        tableWriteConf += Seq(table, keyspace, c) -> conf
-      case _       => tableWriteConf += Seq(table, keyspace) -> conf
-    }
-    this
-  }
-
-  /** Add keyspace level write configuration settings. Set cluster to None for a single cluster */
-  def addKeyspaceLevelWriteConf(keyspace: String, conf: SparkConf, cluster: Option[String]): CassandraSQLContext = {
-    addKeyspaceLevelWriteConf(keyspace, WriteConf.fromSparkConf(conf), cluster)
-  }
-
-  /** Add keyspace level write configuration settings. Set cluster to None for a single cluster */
-  def addKeyspaceLevelWriteConf(keyspace: String, writeConf: WriteConf, cluster: Option[String]) = {
-    cluster match {
-      case Some(c) => validateClusterName(c)
-        keyspaceWriteConf += Seq(keyspace, c) -> writeConf
-      case _       => keyspaceWriteConf += Seq(keyspace) -> writeConf
-    }
-    this
-  }
-
-  /** Add cluster level write configuration settings */
-  def addClusterLevelWriteConf(cluster: String, conf: SparkConf): CassandraSQLContext = {
-    addClusterLevelWriteConf(cluster, WriteConf.fromSparkConf(conf))
-  }
-
-  /** Add cluster level write configuration settings */
-  def addClusterLevelWriteConf(cluster: String, conf: WriteConf) = {
-    validateClusterName(cluster)
-    clusterWriteConf += cluster -> conf
-    this
-  }
-
-  /** Get write configuration settings by the order of table level, keyspace level, cluster level, default settings */
-  def getWriteConf(keyspace: String, table: String, cluster: Option[String]): WriteConf = {
-    cluster match {
-      case Some(c) => validateClusterName(c)
-                      tableWriteConf.get(Seq(table, keyspace, c)).getOrElse(
-                        keyspaceWriteConf.get(Seq(keyspace, c)).getOrElse(
-                          clusterWriteConf.get(c).getOrElse(WriteConf.fromSparkConf(sparkConf))))
-      case _       => tableWriteConf.get(Seq(table, keyspace)).getOrElse(
-                        keyspaceWriteConf.get(Seq(keyspace)).getOrElse(WriteConf.fromSparkConf(sparkConf)))
-    }
-  }
-
-  @transient
-  private val clusterCassandraConnConf = mutable.Map[String, CassandraConnectorConf]()
-
-  /** Add cluster level write configuration settings */
-  def addClusterLevelCassandraConnConf(cluster: String, conf: SparkConf): CassandraSQLContext = {
-    addClusterLevelCassandraConnConf(cluster, CassandraConnectorConf(conf))
-  }
-
-  /** Add cluster level write configuration settings */
-  def addClusterLevelCassandraConnConf(cluster: String, conf: CassandraConnectorConf) = {
-    validateClusterName(cluster)
-    clusterCassandraConnConf += cluster -> conf
-    this
-  }
-
```

Comment:
Oh, actually these methods removed here, has been moved to that other CSQLContext class. Why?